src/eterna/pose2D/Pose2D.ts

```diff
@@ -2840,7 +2838,7 @@ export default class Pose2D extends ContainerObject implements Updatable {
                         ray.fadeIn();
                     }
 
-                    ray.display.position = this.getBaseLoc(ii);
+                    ray.display.position.copyFrom(this.getBaseLoc(ii));
```

Comment:
ok, ignore my previous comments re: set and copyFrom, because obviously copyFrom is just setting the x/y using that other point, not doing some kind of deep copy. whoops